python/tests/core/ecl/test_pylint.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+#  Copyright (C) 2017 Statoil ASA, Norway.
+#
+#  This file is part of ERT - Ensemble based Reservoir Tool.
+#
+#  ERT is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+#  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+#  FITNESS FOR A PARTICULAR PURPOSE.
+#
+#  See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+#  for more details.
+
+import fnmatch
+import os
+import unittest
+from pylint import epylint as lint
+from ert.test import ExtendedTestCase
+
+class CodeTestCase(ExtendedTestCase):
```

Comment:
I guess this file could be split a test class which could go in `python/python/ert/test/code_test.py` and the actual invocation which could go in a normal test class, without explicit `__name__ == '__main__'` code?